### ISSUE TITLE
Set default timeout

### ISSUE THREAD

**Issue by ghost** [androidealp/droide-AjaxProccess#1]
New feature to set a default timeout like:

```
  $(document).ready(function(){
      $('#retornoaqui').droideProcess({getUrl:'acoes.php','timeout':15});
  });
```

this timeout(in seconds) of process in php that is running,
to avoid some process keep running in infinite loop in server.

**Comment by androidealp** [comment]
Ok, I will analyze

**Comment by androidealp** [comment]
I'm insert code timeout.

**Comment by ghost** [comment]
Thanks!  XD 

Resolved: f106db6dbc8ef63bbcb8dbc41eee81694a6db82e
